### ISSUE TITLE
 for training 2x network

### ISSUE THREAD

**Issue by Rainyliuliu** [twtygqyy/pytorch-SRResNet#16]
Hello, for training 2x network ,i change your code in
 self.upscale4x = nn.Sequential(
            nn.Conv2d(in_channels=64, out_channels=256, kernel_size=3, stride=1, padding=1, bias=False),
            nn.PixelShuffle(2),
            nn.LeakyReLU(0.2, inplace=True),
	    #nn.Conv2d(in_channels=64, out_channels=256, kernel_size=3, stride=1, padding=1, bias=False),
            #nn.PixelShuffle(2),
            #nn.LeakyReLU(0.2, inplace=True),
        )

while my test result is similiar with  4x network , could you give me some suggest?
